pkg/renovate/scm.go

```diff
@@ -0,0 +1,58 @@
+package renovate
+
+import (
+	"context"
+
+	"github.com/redhat-appstudio/build-service/pkg/git"
+	"github.com/redhat-appstudio/build-service/pkg/git/credentials"
+)
+
+// Task represents a task to be executed by Renovate with credentials and repositories
+type Task struct {
+	Platform        string
+	Username        string
+	GitAuthor       string
+	RenovatePattern string
+	Token           string
+	Repositories    []*Repository
+}
+
+func AddNewBranchToTheExistedRepositoryTasksOnTheSameHosts(tasks []*Task, component *git.ScmComponent) bool {
```

Comment:
Please add a function description, it's not clear what it does without looking at the code.